src/scss/colors.scss

```
@@ -0,0 +1,10 @@
+$main-color: #3369f3;
+$main-hover: #3369c3;
+$warning-color: #f00;
+$warning-hover: #702017;
+$text-secondary: #999;
+$light-grey: #fbfbfb;
+$dark-grey: #efefef;
+$border-color: #eaeaea;
+$chat-selected: #e4edfd;
+$box-shadow: 0 0 6px rgb(0 0 0 / 14%);
```

HEX тоже позволяет задействовать альфа канал. Можно использовать для единообразия
```suggestion
$box-shadow: 0 0 6px #00000014;
```